package.json

```diff
@@ -32,51 +32,42 @@
   },
   "homepage": "https://github.com/arawa/workspace#readme",
   "devDependencies": {
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/preset-env": "^7.16.11",
+    "@nextcloud/babel-config": "^1.0.0",
     "@nextcloud/browserslist-config": "^2.2.0",
-    "@nextcloud/eslint-config": "^4.0.0",
-    "@nextcloud/eslint-plugin": "^2.0.0",
-    "@nextcloud/webpack-vue-config": "^3.0.0",
+    "@nextcloud/eslint-config": "^6.1.2",
+    "@nextcloud/stylelint-config": "^2.1.2",
+    "@nextcloud/webpack-vue-config": "^4.3.2",
     "@vue/test-utils": "^1.3.0",
-    "babel-eslint": "^10.1.0",
     "babel-loader": "^8.2.3",
-    "chai": "^4.3.4",
-    "css-loader": "^5.2.7",
-    "eslint-webpack-plugin": "^2.6.0",
+    "chai": "^4.3.6",
+    "css-loader": "^6.5.1",
+    "eslint": "^8.8.0",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-webpack-plugin": "^3.1.1",
     "jsdom": "^19.0.0",
     "jsdom-global": "^3.0.2",
-    "mocha": "^9.2.0",
-    "mochapack": "^2.1.2",
+    "linkifyjs": "^3.0.5",
+    "mochapack": "^2.1.4",
+    "node-polyfill-webpack-plugin": "^1.1.4",
     "postcss-html": "^1.3.0",
```

Comment:
I don't think you are using this dependency :thinking:

Reply:
Yes, I use mochapack for tests

However, I don't think I use linkifyjs ^^